tests/helper.js

```diff
@@ -233,52 +221,39 @@ module.exports = {
    * selector : select element
    * item : option(s) in the dropdown.
    */
-  selectDropdownByText: (element, item, milliseconds) => {
-    element.all(by.tagName('option')).then(options => {
-      options.some(option => {
-        option.getText()
-          .then(text => {
-            if (text.indexOf(item) !== -1) {
-              option.click();
-            }
-          })
-          .catch(err => {
-            throw err;
-          });
-      });
-    }).catch(err => {
-      throw err;
-    });
+  electDropdownByText: async (element, item, milliseconds) => {
```

Comment:
Missing the `s` in `selectDropdownByText`. I searched this one too and can't find any references.

Reply:
Fixed!